ci/lint.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 1.22.4-bullseye
```

Comment:
why not latest if the others are?

Reply:
Now that images are being pulled from AWS ECR ... `latest` no longer means latest ... unless someone updates the `latest` in the ECR with the latest.
Also, from time to time the `latest` has broken things and we have had to pin versions ... so pinning is preferred.

Reply:
yeah - I just think we should be consistent.

Reply:
These have now been updated apart from the dp-concourse-tools-nancy image - in which every repo we have "latest" specified as the tag.